README.md

```diff
@@ -120,34 +163,63 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 To run this code, use the following command: 
 <p align="center"> <code> cargo run --release </code> </p>
 
-Note that when running code that uses `tfhe-rs`, it is highly recommended
+> ℹ️ Note that when running code that uses `tfhe-rs`, it is highly recommended
 to run in release mode with cargo's `--release` flag to have the best performances possible.
 
+*This example is explained in more detail in [this part of the documentation](https://docs.zama.ai/tfhe-rs/getting-started/quick_start)*
 
```

Comment:
It's not the same example, so I would say

```
Find a more detailed example in ...
```

Or an example with more explanation in...

Reply:
✅